common/src/main/resources/assets/estrogen/lang/de_de.json

```
@@ -18,50 +24,97 @@
   "attribute.name.estrogen.boob_growing_start_time": "Oberkörper-Startzeit",
   "attribute.name.estrogen.boob_initial_size": "Initiale Oberkörpergröße",
   "attribute.name.estrogen.dash_level": "Dash-Level",
+  "attribute.name.estrogen.fall_damage_resistance": "Fallschadenresistenz",
   "block.estrogen.centrifuge": "Zentrifuge",
   "block.estrogen.cookie_jar": "Keksdose",
   "block.estrogen.cookie_jar.tooltip": "Keksdose",
   "block.estrogen.cookie_jar.tooltip.summary": "Eine Dose zum _aufbewahren_ aller deiner _Kekse_!",
+  "block.estrogen.dormant_dream_block": "Schlafender Traumblock",
+  "block.estrogen.dormant_dream_block.tooltip": "Schlafender Traumblock",
+  "block.estrogen.dormant_dream_block.tooltip.behaviour1": "Während der _Nacht_ §kwird er aktiv",
+  "block.estrogen.dormant_dream_block.tooltip.condition1": "Wenn durch ein Redstone Signal aktiviert",
+  "block.estrogen.dormant_dream_block.tooltip.summary": "Ein Block mit einigen _sehr_ mächtigen Eigenschaften...",
   "block.estrogen.dream_block": "Flasche der Träume",
+  "block.estrogen.estrogen_pill_block": "Östrogen Pillen Schachtel",
```

```suggestion
  "block.estrogen.estrogen_pill_block": "Östrogenpillenschachtel",
```

Again, like "Östrogenpille" - when you now make a box for them, you'd just keep compositing words.